R/calc_views.R

```diff
@@ -19,6 +19,46 @@ calc_views <- function(gsplot){
   
   views <- add_view_labs(views, labs)
   
+  #Start enforcing some order. Background color has to go in back...I'd also put grid in back:
```

Comment:
I feel like we need a strategy for calc_views and some info in the repo's wiki about how to add new features to the package. 
My struggle is that we need to define the new functions in a bunch of different places - config, calc_views (sometimes), and in some cases, overrideGraphics()